### DISCUSSION TITLE
UnitGetUI for MSP

### DISCUSSION THREAD

**Discussion by nolme** [Devolutions/UniGetUI#2268] [General]
Hello,
Whaow. What a great project. :)
- Can we install it using a very silent switch ? (So I can push the application using a RMM software)
- And the next question, once installed, can we run it with an RMM agent (which Service run as SYSTEM) ?
Thanks

**Comment by marticliment** [comment]
Hello,

 - You should be able use the `/verysilent` parameter to call the installer without any window popping up
 - No, WingetUI/UniGetUI does not support installing as a service, and neither does it support being used as a software deployer tool

**Reply by nolme** [reply]
ok thanks for the so quick answer.
